agent/bin/ci.sh

```diff
@@ -36,6 +36,8 @@ pub get
 
 dart bin/agent.dart ci -c "$CONFIG_FILE" 2>&1 >> "$LOG_FILE" &
 
+disown
```

Comment:
I'd put this at the end of the previous line, after the "&", so it's clear what it applies to.